### ISSUE TITLE
How to set the learning rate decay in TFF?

### ISSUE THREAD

**Issue by Chun78HunZi** [tensorflow/federated#1528]

**Comment by zcharles8** [comment]
Hi @Chun78HunZi. This seems like a question that is better suited to the `tensorflow-federated` tag on Stack Overflow.  The issues tab here is for things like feature requests and bug reporting, rather than to provide guidance on TFF usage. Would you mind asking it there, with some more accompanying details about exactly what you're trying to do?
